src/motors.c

```diff
@@ -56,6 +150,33 @@ extern void init_motors()
 /* Robert's extern/visible functions go below here: */
 extern void motor_set_speed_percent(motors_t motor_x, uint8_t speed, direction_t dir)
 {
+	if(dir == Reverse) speed *= -1;
```

Comment:
So 9% duty cycle is neutral, 6% is full reverse and 12% is full forward. Thus we need to make reverse and forward reflect this.